### DISCUSSION TITLE
IOS Native views

### DISCUSSION THREAD

**Discussion by Varmod** [AvaloniaUI/Avalonia#16194] [Q&A] [answered]
Hi, does Avalonia support native views on IOS? I've found information on [Android](https://docs.avaloniaui.net/docs/guides/platforms/android/embed-native-views) in the docs but can't seem to get a definite answer for IOS.

Thanks!

**Comment by maxkatz6** [comment, marked as answer]
Yes, it's done very similarly https://github.com/AvaloniaUI/Avalonia/blob/master/samples/ControlCatalog.iOS/EmbedSample.iOS.cs
